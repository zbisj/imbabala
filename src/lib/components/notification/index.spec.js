import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Notification from ".";

describe("Notification", () => {
  test("renders correctly", () => {
    const tree = renderer
      .create(
        <Notification
          variant="primary"
          heading="heading"
          content="content"
          alignHeading="center"
          alignContent="center"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
