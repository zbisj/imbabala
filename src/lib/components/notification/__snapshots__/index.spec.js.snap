// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification renders correctly 1`] = `
<div
  className=""
>
  <h3
    className=""
  >
    heading
  </h3>
  <p
    className=""
  >
    content
  </p>
</div>
`;
